<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSimulationDialog</class>
 <widget class="QDialog" name="NewSimulationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>new simulation dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <property name="spacing">
    <number>16</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>physical properties</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>34</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>width</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>64</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>height</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sizeXEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>400</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="sizeYEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>200</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="simulationParametersButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>simulation parameters</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>34</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>energy</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="energyEdit">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string>metadata</string>
     </property>
     <widget class="QPushButton" name="symbolTableButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>28</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>symbol table</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewSimulationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewSimulationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
